checks.nix

```
@@ -32,13 +32,8 @@ in
       };
     };
   }).activationPackage;
-  init-example-el = self.outputs.package.${system} {
-    doomPrivateDir = ./test/doom.d;
-    dependencyOverrides = inputs;
-  };
-  init-example-el-emacsGit = self.outputs.package.${system} {
-    doomPrivateDir = ./test/doom.d;
-    dependencyOverrides = inputs;
+  init-example-el = self.outputs.packages.${system}.doom-emacs;
```

It's just about screaming for a `let..in` def :P
```suggestion
  init-example-el = doom;
```

---

I particularly prefer using `inherit` here, so this is what I went instead.